packages/braid-design-system/src/lib/components/useToast/useTimeout.ts

```diff
@@ -15,6 +15,7 @@ export const useTimeout = ({ onTimeout, duration }: UseTimeoutProps) => {
 
   useEffect(() => {
     if (activated) {
+      setActivated(true);
```

Comment:
One method of having the toast work on the docs site in local dev. Was hitting an issue with strict mode

Reply:
Might need a catch up on this one 😕